__tests__/registries/npm-registry.js

```diff
@@ -175,6 +175,24 @@ describe('request', () => {
 
     expect(requestParams.headers.authorization).toBe('Bearer testAuthToken');
   });
+
+  test('should add authorization header with token for scope if pathname is to registry and is scoped package', () => {
```

Comment:
Let's explicitly say "custom registry" in the test name:
`should add authorization header with token for custom registries with a scoped package`